controllers/consoleplugin/config/static-frontend-config.yaml

```diff
@@ -790,6 +797,15 @@ filters:
     name: ICMP code
     component: number
     hint: Specify an ICMP code value as integer number.
+  - id: tcp_flags
+    name: TCP flags
+    component: number
```

Comment:
Should we add an autocomplete here to display strings as:
```
    FIN_FLAG
    SYN_FLAG
    RST_FLAG
    PSH_FLAG
    ACK_FLAG
    URG_FLAG
    ECE_FLAG
    CWR_FLAG
    SYN_ACK_FLAG
    FIN_ACK_FLAG
    RST_ACK_FLAG
```

You can check Protocol, DSCP and ICMP fields as examples:
https://github.com/netobserv/network-observability-console-plugin/blob/main/web/src/utils/filter-definitions.ts#L300-L302
https://github.com/netobserv/network-observability-console-plugin/blob/main/web/src/utils/filter-options.ts#L164-L170
https://github.com/netobserv/network-observability-console-plugin/blob/main/web/src/utils/dscp.ts#L7